### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by rickducott**

### CODE REVIEW

**Comment by rickducott** [comment]
This should finally be done.

File: cli/cmd/main.go [hunk 1]
```diff
@@ -17,7 +17,7 @@ func main() {
 	app := cmd.App(version.Version)
 	if err := app.Execute(); err != nil {
 		if err != nil {
-			panic(err)
+			os.Exit(1)
```
Comment (EItanya) [inline cli/cmd/main.go]: I kinda like the way palantir does this. https://github.com/palantir/bulldozer/blob/develop/main.go
Not for right now, but worth looking at

**Comment by rickducott** [comment]
Tested install and uninstall via command line

```
make manifest TAGGED_VERSION=v0.2.2
make sqoopctl
_output/sqoopctl install kube -f _output/helm/charts/sqoop-0.2.2.tgz
_output/sqoopctl uninstall
```

File: cli/pkg/cmd/install/root.go [hunk 2]
```diff
@@ -29,12 +38,35 @@ func UninstallCmd(opts *options.Options, optionsFunc ...cliutils.OptionsFunc) *c
 		Short: constants.UNINSTALL_COMMAND.Short,
 		Long:  constants.UNINSTALL_COMMAND.Long,
 		RunE: func(cmd *cobra.Command, args []string) error {
-			if err := deleteNamespace(installNamespace); err != nil {
-				return errors.Wrapf(err, "delete sqoop failed")
+			fmt.Printf("Uninstalling Sqoop. This might take a while...")
```
Comment (EItanya) [inline cli/pkg/cmd/install/root.go]: Should we maybe warn that we are deleting the entire namespace? I know we do this elsewhere as well, but it seems like people might have resources they don't want deleted.

**Pull request merged**